publiccode.yml

```diff
@@ -0,0 +1,40 @@
+publiccodeYmlVersion: "0"
```

Comment:
I guess this needs to be `0.4`
https://yml.publiccode.tools/schema.core.html#key-publiccodeymlversion